modules/process/manager.go

```diff
@@ -6,13 +6,12 @@ package process
 
 import (
 	"bytes"
+	"context"
```

Comment:
I don't agree that strictly supporting old toolchains would reward the project's developers and users that much, if any. Just speaking for my own opinion though. ❤️